### DISCUSSION TITLE
Fluent DataGrid "No data to show!"

### DISCUSSION THREAD

**Discussion by satyajit-behera** [microsoft/fluentui-blazor#2700] [Q&A]
Hi,
Is it possible to customize the message  "No data to show!" while working with FluentDataGrid and it does not have any data to show?
Thanks

**Comment by vnbaaij** [comment]
Yes: 
![image](https://github.com/user-attachments/assets/0c81fe38-d9e0-4d46-be71-5207b6af6bc3)


See https://www.fluentui-blazor.net/DataGrid#customcomparer

**Reply by barruka** [reply]
In the url provided: https://www.fluentui-blazor.net/DataGrid#customcomparer
It is not specified how to change the text of the message "No data to show!"

Could you please improve the doc at least?

In the section where the parameter EmptyContent is detailed, would be nice to upgrade the text:
`If specified, grids render this fragment when there is no content.`
to
`If specified, grids render this fragment when there is no content, instead of the "No data to show!" default message.`

**Reply by vnbaaij** [reply]
Not every sample handles all parameters.

The EmptyContent (and LoadingContent) parameters have their own sample page.. https://www.fluentui-blazor.net/datagrid-loading-and-empty-content
